tests/test_hesiod.py

```diff
@@ -26,11 +42,46 @@ def test() -> None:
     test()
 
 
+def test_load_config_complex() -> None:
+    cwd = Path(".").absolute()
+    run_cfg_file = cwd / "tests/runs/run_complex.yaml"
+    base_cfg_dir = cwd / "tests/cfg"
```

Comment:
I see these lines repeated in every test. You can use [pytest fixtures](https://docs.pytest.org/en/stable/fixture.html) for this use case